.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
+!test/fixtures/loader/node_modules/
```

Comment:
Why  is the `node_modules` directory under the `loader` directory not ignored?

Reply:
They are test fixtures. The directory must be named `node_modules` to adhere to how the Node module loader looks for modules.

Reply:
Ok that makes sense.